src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -287,8 +293,25 @@ public void render(Graphics g)
     // Draw the Commander overlay with available funds.
     drawHUD(screenGraphics, showTileDetails);
 
-    // Copy the screen image into the window's graphics buffer.
-    g.drawImage(screenImage, 0, 0, screenImage.getWidth()*drawScale, screenImage.getHeight()*drawScale, null);
+    return screenImage;
```

Comment:
Might make sense to just return the map image itself here, and handle putting it in the right place elsewhere.

Reply:
It might. I thought about that, and didn't, because by then it was working, and I try to avoid fixing stuff that works.